src/Datadog.Trace.ClrProfiler.Managed.Loader/Startup.cs

```diff
@@ -1,52 +1,186 @@
 using System;
+using System.IO;
 using System.Reflection;
 
 namespace Datadog.Trace.ClrProfiler.Managed.Loader
 {
     /// <summary>
-    /// A class that attempts to load the Datadog.Trace.ClrProfiler.Managed .NET assembly.
+    /// This is the only publi class in this assembly.
```

Comment:
```suggestion
    /// This is the only public class in this assembly.
```

Reply:
Updated in _macrogreg/loader-with-modules-cctor-and-managed-loader_-branch.